benchmarks/benchmarks.py

```diff
@@ -0,0 +1,66 @@
+# # Write the benchmarking functions here.
+# # See "Writing benchmarks" in the asv docs for more information.
+
```

Comment:
Why is this a commented out file?

Reply:
Yep, rather that than just comment out

Reply:
Done in https://github.com/tardis-sn/tardis/pull/2260/commits/55d13d1ba920bcd8bd1867f296c4d9a9a2537c6d